lib/tmuxinator/project.rb

```diff
@@ -381,5 +381,43 @@ def parsed_parameters(parameters)
     def wemux?
       yaml["tmux_command"] == "wemux"
     end
+
+    def enable_pane_titles?
+      yaml["enable_pane_titles"]
+    end
+
+    def tmux_set_pane_title_position
+      if pane_title_position?
+        "#{tmux} set pane-border-status #{yaml['pane_title_position']}"
+      else
+        "#{tmux} set pane-border-status top"
+      end
+    end
+
+    def pane_title_position?
+      yaml["pane_title_position"]
+    end
+
+    def tmux_set_pane_title_format
+      if pane_title_format?
+        "#{tmux} set pane-border-format \"#{yaml['pane_title_format']}\""
+      else
+        "#{tmux} set pane-border-format " + '"#{pane_index}: #{pane_title}"'
```

Comment:
I haven't tested this, but [code climate is complaining about the interpolation within single quotes](https://codeclimate.com/github/tmuxinator/tmuxinator/pull/892), which makes sense!

Reply:
Ya that can't be helped since tmux use the same text formating as ruby. Unless you know a way?

Reply:
You could try "interpolable" string literals: https://docs.ruby-lang.org/en/master/syntax/literals_rdoc.html#label-25+and+-25Q-3A+Interpolable+String+Literals

Reply:
hmm, not sure what you mean. I attempted it in my latest commit, but it's still rejected. Maybe I did it wrong though.

Reply:
ok figured something out